File: testapp/src/main/java/com/example/testapp/BookMessage.java
package com.example.testapp;

/**
 * Author : AqrLei
 * Name : MyLearning
 * Description :
 * Date : 2017/11/10.
 */

public class BookMessage {
    private String name;
    private String path;
    private String encoding;
    private long accessTime;


    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    public String getEncoding() {
        return encoding;
    }

    public void setEncoding(String encoding) {
        this.encoding = encoding;
    }

    public long getAccessTime() {
        return accessTime;
    }

    public void setAccessTime(long accessTime) {
        this.accessTime = accessTime;
    }
}

File: yueting/src/main/kotlin/com/aqrlei/graduation/yueting/model/local/BookMessage.kt
package com.aqrlei.graduation.yueting.model.local

import android.graphics.drawable.Drawable
import java.io.Serializable

/**
 * @Author: AqrLei
 * @Name MyLearning
 * @Description:
 * @Date: 2017/8/28
 */
data class BookMessage(
        var name: String = "",
        var bookIcon: Drawable? = null,
        var path: String = "",
        var encoding: String = "",
        var accessTime: Long = 0
) : Serializable {
    override fun equals(other: Any?): Boolean {
        if (other is BookMessage) {

            return other.name == this.name && other.path == this.path
        }
        return super.equals(other)
    }
}